annotation-processing/src/test/resources/test/fields/setter/InvalidStudent.java

```diff
@@ -8,6 +8,7 @@
 
     private String name;
 
+    @SuppressWarnings({"methodName"})
```

Comment:
If I'm not mistaken it can be written in form `checkstyle:methodName`. If it can, please change it. It helps to see what exactly this `@SuppresWarnings` disables